src/api/tx_initiation/builder/transaction_details_builder.rs

```diff
@@ -261,7 +278,10 @@ impl TransactionDetailsBuilder {
                 .mutator_set_accumulator_after()
                 .expect("Block from state must have mutator set after"),
```

Comment:
This line is violating rule 3 in src/api/README-dev.md:

> 3. no unwrap(), expect(), or panic!() in this layer.

there should be a CreateTxError variant for this, and thus it should simply be:

```
tip_block.mutator_set_accumulator_after()?
```